src/hotspot/share/classfile/verifier.hpp

```diff
@@ -52,7 +52,7 @@ class Verifier : AllStatic {
   // Return false if the class is loaded by the bootstrap loader,
   // or if defineClass was called requesting skipping verification
   // -Xverify:all overrides this value
```

Comment:
This comment doesn't describe the new operation of this method.

Reply:
You're right.  There was an old change that made defineClass explicitly disable verification.  That code is gone now.